libs/visualizer-connector/README.md

```diff
@@ -0,0 +1,193 @@
+# DIAL Visualizer Connector
+
+DIAL Visualizer Connector is a library for connecting custom visualizers - applications which could visualize some special type data (for example **plot data** for the **Plotly**).
+
+## Public classes to use
+
+`VisualizerConnector` - class which creates iframe with provided **VisualizerConnector**, allows to interact with **Visualizer** rendered in the iframe (send/receive messages). Types for configuration options is `VisualizerConnector`.
+
+## Prerequisites
+
+Your Dial Chat application should configure hosts where your custom visualizers hosted. You could do this with `ALLOWED_IFRAME_SOURCES` env variable
+
+\_Note: For development purposes you can set `*`\_
+
+```
+ALLOWED_IFRAME_SOURCES=http://localhost:8000
+```
+
+Moreover, it needs to be configured some **Visualizer** properties:
+
+- `CUSTOM_VISUALIZERS_TYPES` - list of the attachment content types for which you want to use custom visualizers.
+
+```
+CUSTOM_VISUALIZERS_TYPES='<custom_type_1>, <custom_type_2>'
+```
```

Comment:
I didn't understand for what do we need this list.
`CUSTOM_VISUALIZERS` already have `ContentType` property.